regexec.c

```diff
@@ -11158,7 +11158,7 @@ S_reghop3(U8 *s, SSize_t off, const U8* lim)
                 while (s > lim && UTF8_IS_CONTINUATION(*s))
                     s--;
                 if (! UTF8_IS_START(*s)) {
-                    Perl_croak_nocontext("Malformed UTF-8 character (fatal)");
+                    croak("Malformed UTF-8 character (fatal)");
```

Comment:
these are close to panics, but @khwilliamson would be better suited to commenting